tasks/dataset-split/src/sequenceDatasetSplit.py

```diff
@@ -0,0 +1,110 @@
+from pathlib import Path
+
+import csv
+import logging
+import zipfile
+
+from coretex import currentTaskRun, folder_manager, SequenceDataset, NetworkDataset, CustomDataset, SequenceSample
+
+import chardet
+
+from .utils import splitOriginalSamples
+
+
+CASEINSENSITIVE_NAMES = ["id", "sampleid", "sample id", "sample-id", "featureid" ,"feature id", "feature-id", "sample_id", "sample.id"]
+CASESENSITIVE_NAMES = ["SampleID" , "Sample ID", "OTUID", "OTU ID", "sample_name"]
+
+
+class MetadataFileNotFoundError(Exception):
+    def __init__(self, *args: object) -> None:
+        super().__init__(*args)
```

Comment:
Since this adds no new functionality for this exception remove __init__ override and replace it with `pass` keyword in the body of the class